BikeBattle/app/src/main/java/edu/hm/cs/bikebattle/app/activities/BaseActivity.java

```diff
@@ -140,4 +141,6 @@ public Uri getUserPhoto(){
   public DataConnector getDataConnector() {
     return dataConnector;
   }
+
+  public abstract  void refreshUserInfo();
```

Comment:
Gute Idee, aber ich würde es nicht abstract machen, da man sonst die Methode überschreiben MUSS. Das macht z.B. in der LoginView null sinn, da es hier nie einen User geben wird.